docs/user-guide/api-mediation/using-multi-factor-authentication.md

```diff
@@ -21,11 +21,7 @@ If you use z/OSMF as your authentication provider, ensure that you meet the foll
 When z/OSMF is used as a security provider, it is possible to reuse MFA tokens, whereby it is possible to receive a JWT token based on previously used MFA token. This presents a security risk.  
 
 This issue can be resolved by configuring z/OSMF to work properly with API ML.
-Update the z/OSMF configuration with the following parameter:
-`allowBasicAuthLookup="false"`
-
-After applying this change, each authentication call results in generating a new JWT.
-For more information, see [Configuring z/OSMF](../systemrequirements-zosmf.md) to properly work with API ML. 
+Follow the instructions in [Configuring z/OSMF to properly work with API ML](../systemrequirements-zosmf.md#disable-cache-in-zosmf) to make the changes to z/OSMF settings.
```

Comment:
I cannot find this anchor link in the article Configuring z/OSMF. Can you please specify the section that needs to be configured or can we link to the main article?